waspc/ChangeLog.md

```diff
@@ -3,12 +3,28 @@
 ## Unreleased
 
 ### ⚠️ Breaking Changes
+Address all breaking changes by following [the official migration guide](https://wasp.sh/docs/migration-guides/migrate-from-0-16-to-0-17). This is just a short overview:
 
 - In the `usernameAndPassword` authentication method, the function `login()` imported from `wasp/client/auth` now accepts an object with `username` and `password` instead of two separate arguments ([#2598](https://github.com/wasp-lang/wasp/pull/2598))
+- We've made some improvements to our TypeScript setup that require you to update the `tsconfig.json` file. The migration guide will lead you through them.
+  For posterity, here are all the non-cosmetic changes we've made:
+  ```json
+  {
+    "compilerOptions": {
+      // ...
+      "moduleDetection": "force",
+      "isolatedModules": true,
+      // Remove 'typeRoots'
+    },
+  }
+  ```
+- [The issue](https://github.com/testing-library/jest-dom/issues/546#issuecomment-1889884843) with `jest-dom` types is now addressed in `src/vite-env.d.ts`.
```

Comment:
This entry is not very understandable for users I think, especially since it links to an issue in another project altogether, and the error it describes is not one our users were having (because we were already solving it). 

I'd go for something along the lines of

```suggestion
- The types for DOM helpers in tests are now imported from `src/vite-env.d.ts` instead of changing the `tsconfig.json`'s `typeRoots`.
```

(and I'm sure it can be improved further 😅)